app-frontend/src/app/components/projects/aoiDrawToolbar/index.html

```diff
@@ -1,95 +1,82 @@
-<div class="navbar light-navbar"
-     ng-style="$ctrl.eleStyle">
-  <div class="navbar-section aoi-draw-toolbar-left"
-       ng-style="$ctrl.leftStyle">
-    <button class="btn btn-small btn-primary btn-aoi-actions"
-            ng-click="$ctrl.onDrawAoi()"
-            ng-disabled="$ctrl.isDrawingAoi"
-            ng-if="!$ctrl.isValidGeometry()">
-      <i class="icon-plus"></i> Draw AOI
-    </button>
-    <button class="btn btn-small btn-primary btn-aoi-actions"
-            ng-if="$ctrl.isValidGeometry()"
-            ng-disabled="$ctrl.isEditingAoi"
+<div class="navbar light-navbar" ng-style="$ctrl.eleStyle">
+    <div class="navbar-section aoi-draw-toolbar-left" ng-style="$ctrl.leftStyle">
+        <button class="btn btn-tiny btn-transparent" ng-click="$ctrl.onClickCancel()">
```

Comment:
I think it would be good if we disabled the back button if there is any map/shape operations in progress. Similar to what we did to the old `Cancel` button.